rspec/rspec.gemspec

```diff
@@ -16,10 +16,10 @@ Gem::Specification.new do |s|
   s.description = "BDD for Ruby"
 
   s.metadata = {
-    'bug_tracker_uri' => 'https://github.com/rspec/rspec-metagem/issues',
+    'bug_tracker_uri' => 'https://github.com/rspec/rspec/issues',
     'documentation_uri' => 'https://rspec.info/documentation/',
     'mailing_list_uri' => 'https://groups.google.com/forum/#!forum/rspec',
-    'source_code_uri' => 'https://github.com/rspec/rspec',
+    'source_code_uri' => "https://github.com/rspec/rspec/tree/v#{s.version}",
```

Comment:
This is not good but we do not have a tag yet. We could redirect to maintenance branch but it could still be wrong. What about having a tag for that?

Reply:
`[rspec-metagem-v3.13.0](https://github.com/rspec/rspec/releases/tag/rspec-metagem-v3.13.0)`